mosec/server.py

```diff
@@ -324,6 +328,7 @@ def append_worker(
         max_wait_time: int = 0,
         start_method: str = "spawn",
         env: Union[None, List[Dict[str, str]]] = None,
+        timeout: int = 3,
```

Comment:
Should the default value be the `--timeout`?

Reply:
--timeout isn't a list. Workers of different stages could have different timeout values due to different complexity.

Reply:
I mean the default value. It should be the same as the parameter unless user specifies another one. Otherwise, the service will always timeout for stable diffusion.

Reply:
So I set default timeout = 0 (invalid), if user doesn't specify it, it will be changed to self._configs["timeout"].
```python
# timeout: int = 0
self._worker_timeout.append(
            timeout if timeout >= 1 else self._configs["timeout"] // 1000 # milliseconds to second
        )
```